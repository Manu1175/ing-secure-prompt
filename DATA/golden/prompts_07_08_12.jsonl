{"id": "prompts_07_08_12:0", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft an ops email confirming transfer 490d1104-7e77-4bf6-9827-4022c36735a3 completed on 2024-11-18 for EUR 1,173.73 to IBAN BE96 3296 4713 6802.", "expected_scrub": "Draft an ops email confirming transfer <TRANSFER_ID> completed on <DATE> for <CURRENCY> <AMOUNT> to IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:1", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Open a failure investigation for transfer f9863527-0b1a-48b3-917b-03aca2d74ba3 (EUR 5,032.24) to IBAN BE76 1541 7848 7143 with Status: Failed on 2025-06-12.", "expected_scrub": "Open a failure investigation for transfer <TRANSFER_ID> (<CURRENCY> <AMOUNT>) to IBAN <IBAN> with Status: <STATUS> on <DATE>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:2", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a reconciliation note for Pending transfers: be41f293-ae1b-458e-b9d1-5266037b9332 (EUR 9,428.45 to BE14 8365 4959 2937), fc722f2f-3bb1-4fb1-96f7-d517e4638697 (EUR 746.86 to BE34 3730 4312 2754), and 15705bcc-2ae3-4f0d-8968-03fa0b55ef77 (EUR 4,038.46 to BE49 3124 1282 1010).", "expected_scrub": "Prepare a reconciliation note for <STATUS> transfers: <TRANSFER_ID> (<CURRENCY> <AMOUNT> to <IBAN>), <TRANSFER_ID> (<CURRENCY> <AMOUNT> to <IBAN>), <TRANSFER_ID> (<CURRENCY> <AMOUNT> to <IBAN>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:3", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Send a risk alert for high-value Pending transfer ff2fcf47-4341-4729-86cd-ee1db7853be5 (EUR 9,573.90) to IBAN BE33 3644 9891 9701.", "expected_scrub": "Send a risk alert for high-value <STATUS> transfer <TRANSFER_ID> (<CURRENCY> <AMOUNT>) to IBAN <IBAN>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:4", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Create an SLA snapshot listing: 490d1104-7e77-4bf6-9827-4022c36735a3 (Completed), e092a2a7-a5d7-4006-bdaa-91fb64592a79 (Completed), f9863527-0b1a-48b3-917b-03aca2d74ba3 (Failed), 25f7d032-14c1-4ce4-bc8b-7da2a9855be1 (Pending).", "expected_scrub": "Create an SLA snapshot listing: <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:5", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Flag IBAN BE75 9060 6135 7217 for review after Failed transfer 4ce0acc6-1ce9-4040-874b-8716df38ea53 (EUR 311.10) on 2025-08-28.", "expected_scrub": "Flag IBAN <IBAN> for review after <STATUS> transfer <TRANSFER_ID> (<CURRENCY> <AMOUNT>) on <DATE>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:6", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Standup note: Completed transfers 36d0b951-2d3d-4161-abff-147784fb9e6d (BE20 6428 3022 9164) and d45765d5-1f91-4c88-a067-c4e531654c97 (BE11 7045 8358 2117); Pending 798d09c1-2142-4dd5-b69a-7d50f3bb3087 (BE96 6508 4858 2913).", "expected_scrub": "Standup note: Completed transfers <TRANSFER_ID> (<IBAN>) and <TRANSFER_ID> (<IBAN>); Pending <TRANSFER_ID> (<IBAN>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:7", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Treasury forecast: include Pending amounts be41f293-ae1b-458e-b9d1-5266037b9332 EUR 9,428.45; 15705bcc-2ae3-4f0d-8968-03fa0b55ef77 EUR 4,038.46; 2fc89765-a638-4b6a-881a-83b5cfa4b877 EUR 2,361.18.", "expected_scrub": "Treasury forecast: include Pending amounts <TRANSFER_ID> <CURRENCY> <AMOUNT>; <TRANSFER_ID> <CURRENCY> <AMOUNT>; <TRANSFER_ID> <CURRENCY> <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:8", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a customer notification for transfer 0fa03d49-b3ad-4013-a30a-19d73b8ce45b Completed to IBAN BE65 4261 5143 8013 (EUR 4,953.02).", "expected_scrub": "Draft a customer notification for transfer <TRANSFER_ID> <STATUS> to IBAN <IBAN> (<CURRENCY> <AMOUNT>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:9", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Provide a compliance sample set: 03488215-4c4a-49aa-ae1c-b7f6ede56a4a (Pending), 62a4a672-afa3-4c18-b13a-3af54efc40b5 (Pending), 093263? (use 09263263-d59e-4409-98f6-cf0da2422992 Completed), 8b0b5274-7fff-444a-ab6f-deccbb5ffbd4 (Completed), 9f4f8842-ff31-42ae-b144-08b66b49093c (Failed).", "expected_scrub": "Provide a compliance sample set: <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>), <TRANSFER_ID> (<STATUS>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:10", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a decommission plan for Retired physical apps: PhysicalApp_1 (VM-PH-1, DOC E4TJ7K), PhysicalApp_2 (VM-PH-2, DOC SGBBFX), PhysicalApp_11 (VM-PH-11, DOC 60V5MC), PhysicalApp_13 (VM-PH-13, DOC PWKE24), PhysicalApp_17 (VM-PH-17, DOC 82L5I1), PhysicalApp_18 (VM-PH-18, DOC IGBRAZ), PhysicalApp_21 (VM-PH-21, DOC L2VCXO), PhysicalApp_26 (VM-PH-26, DOC UL1QUW).", "expected_scrub": "Draft a decommission plan for <OP_STATUS> physical apps: <APP_NAME> (<VM_NAME>, DOC <DOC_REF>), ...", "entities": null, "notes": null}
{"id": "prompts_07_08_12:11", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Propose a maintenance window for Production apps in Maintenance: PhysicalApp_3 (8 vCPU, 8 GB; DOC FGPAA4), PhysicalApp_4 (4 vCPU, 16 GB; DOC 4MIYVZ), PhysicalApp_5 (4 vCPU, 32 GB; DOC N4DBRS), PhysicalApp_6 (8 vCPU, 8 GB; DOC 3W7MGK), PhysicalApp_8 (2 vCPU, 32 GB; DOC PPDXJX).", "expected_scrub": "Propose a maintenance window for <ENVIRONMENT> apps in <OP_STATUS>: <APP_NAME> (<VCPU_RAM>; DOC <DOC_REF>), ...", "entities": null, "notes": null}
{"id": "prompts_07_08_12:12", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a capacity snapshot for Running/Production: PhysicalApp_19 (Storage 1261 GB; 2 vCPU, 8 GB), PhysicalApp_25 (1890 GB; 8 vCPU, 16 GB), PhysicalApp_27 (1180 GB; 4 vCPU, 8 GB), PhysicalApp_30 (991 GB; 4 vCPU, 16 GB).", "expected_scrub": "Create a capacity snapshot for <OP_STATUS>/<ENVIRONMENT>: <APP_NAME> (Storage <STORAGE_GB> GB; <VCPU_RAM>), ...", "entities": null, "notes": null}
{"id": "prompts_07_08_12:13", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Write a cost model note proposing OpEx migration for Production apps PhysicalApp_25 (VM-PH-25) and PhysicalApp_30 (VM-PH-30), currently CapEx with Internal IT provider.", "expected_scrub": "Write a cost model note proposing OpEx migration for <ENVIRONMENT> apps <APP_LIST> (current <COST_MODEL> with <PROVIDER>).", "entities": null, "notes": null}
{"id": "prompts_07_08_12:14", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Map environments: Development — PhysicalApp_1 (Retired), _9 (Maintenance), _14 (Maintenance), _23 (Running), _28 (Maintenance); Test — PhysicalApp_7 (Maintenance), _18 (Retired), _20 (Maintenance), _21 (Retired), _24 (Running), _26 (Retired), _29 (Running); Production — multiple in Maintenance/Running as listed.", "expected_scrub": "Map environments: Development — <APP_LIST>; Test — <APP_LIST>; Production — <SUMMARY>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:15", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft an activation email for customer agreement CUST-0023 (Line of Credit Agreement) for Emily Williams (emily.williams@hotmail.com), Status Active, Amount 353,236.74.", "expected_scrub": "Draft an activation email for customer agreement <AGREEMENT_ID> (<AGREEMENT_TYPE>) for <PARTY_NAME> (<PARTY_EMAIL>), Status <STATUS>, Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:16", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Create a termination summary for CUST-0002 (Letter of Credit) — Michael Clark (michael.clark@gmail.com), Status Terminated, Amount 499,667.34.", "expected_scrub": "Create a termination summary for <AGREEMENT_ID> (<AGREEMENT_TYPE>) — <PARTY_NAME> (<PARTY_EMAIL>), Status <STATUS>, Amount <AMOUNT>.", "entities": null, "notes": null}
{"id": "prompts_07_08_12:17", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "List active outsourcing/related supplier agreements: SUP-0012 (Outsourcing Agreement) 64,145.18; SUP-0036 (Outsourcing Agreement) 396,361.85; SUP-0039 (Outsourcing Agreement) 61,399.13; SUP-0034 (Services Supply Agreement) 237,777.45; SUP-0024 (Goods Supply Agreement) 106,623.59.", "expected_scrub": "List active supplier agreements of specified types with IDs and Amounts: <AGREEMENT_ID> (<TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_07_08_12:18", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Prepare a pending-action tracker for customers: CUST-0044 (Letter of Credit) 310,738.37; CUST-0045 (Bank Guarantee) 212,569.00; CUST-0046 (Personal Loan Agreement) 451,133.43; CUST-0054 (Safe Deposit Box Agreement) 203,538.70; CUST-0055 (Credit Card Agreement) 279,479.12; CUST-0056 (Overdraft Agreement) 492,879.62.", "expected_scrub": "Prepare a pending-action tracker for customers: <AGREEMENT_ID> (<AGREEMENT_TYPE>) <AMOUNT>; ...", "entities": null, "notes": null}
{"id": "prompts_07_08_12:19", "source_path": "external/SecurePrompt/PROMPTS/prompts_07_08_12.xlsx", "modality": "text", "c_level": null, "raw_text": "Draft a risk note flagging high amounts: CUST-0028 (Active) 497,139.00; SUP-0016 (Expired) 496,703.00; SUP-0038 (Expired) 497,735.94; CUST-0002 (Terminated) 499,667.34.", "expected_scrub": "Draft a risk note flagging high amounts: <AGREEMENT_ID> (<STATUS>) <AMOUNT>; ...", "entities": null, "notes": null}
