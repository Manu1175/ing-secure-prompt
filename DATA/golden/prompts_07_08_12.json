[
  {
    "text": "Draft an ops email confirming transfer 490d1104-7e77-4bf6-9827-4022c36735a3 completed on 2024-11-18 for EUR 1,173.73 to IBAN BE96 3296 4713 6802.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Open a failure investigation for transfer f9863527-0b1a-48b3-917b-03aca2d74ba3 (EUR 5,032.24) to IBAN BE76 1541 7848 7143 with Status: Failed on 2025-06-12.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Prepare a reconciliation note for Pending transfers: be41f293-ae1b-458e-b9d1-5266037b9332 (EUR 9,428.45 to BE14 8365 4959 2937), fc722f2f-3bb1-4fb1-96f7-d517e4638697 (EUR 746.86 to BE34 3730 4312 2754), and 15705bcc-2ae3-4f0d-8968-03fa0b55ef77 (EUR 4,038.46 to BE49 3124 1282 1010).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Send a risk alert for high-value Pending transfer ff2fcf47-4341-4729-86cd-ee1db7853be5 (EUR 9,573.90) to IBAN BE33 3644 9891 9701.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create an SLA snapshot listing: 490d1104-7e77-4bf6-9827-4022c36735a3 (Completed), e092a2a7-a5d7-4006-bdaa-91fb64592a79 (Completed), f9863527-0b1a-48b3-917b-03aca2d74ba3 (Failed), 25f7d032-14c1-4ce4-bc8b-7da2a9855be1 (Pending).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Flag IBAN BE75 9060 6135 7217 for review after Failed transfer 4ce0acc6-1ce9-4040-874b-8716df38ea53 (EUR 311.10) on 2025-08-28.",
    "labels": [
      "DOB",
      "PHONE"
    ]
  },
  {
    "text": "Standup note: Completed transfers 36d0b951-2d3d-4161-abff-147784fb9e6d (BE20 6428 3022 9164) and d45765d5-1f91-4c88-a067-c4e531654c97 (BE11 7045 8358 2117); Pending 798d09c1-2142-4dd5-b69a-7d50f3bb3087 (BE96 6508 4858 2913).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Treasury forecast: include Pending amounts be41f293-ae1b-458e-b9d1-5266037b9332 EUR 9,428.45; 15705bcc-2ae3-4f0d-8968-03fa0b55ef77 EUR 4,038.46; 2fc89765-a638-4b6a-881a-83b5cfa4b877 EUR 2,361.18.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a customer notification for transfer 0fa03d49-b3ad-4013-a30a-19d73b8ce45b Completed to IBAN BE65 4261 5143 8013 (EUR 4,953.02).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Provide a compliance sample set: 03488215-4c4a-49aa-ae1c-b7f6ede56a4a (Pending), 62a4a672-afa3-4c18-b13a-3af54efc40b5 (Pending), 093263? (use 09263263-d59e-4409-98f6-cf0da2422992 Completed), 8b0b5274-7fff-444a-ab6f-deccbb5ffbd4 (Completed), 9f4f8842-ff31-42ae-b144-08b66b49093c (Failed).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a decommission plan for Retired physical apps: PhysicalApp_1 (VM-PH-1, DOC E4TJ7K), PhysicalApp_2 (VM-PH-2, DOC SGBBFX), PhysicalApp_11 (VM-PH-11, DOC 60V5MC), PhysicalApp_13 (VM-PH-13, DOC PWKE24), PhysicalApp_17 (VM-PH-17, DOC 82L5I1), PhysicalApp_18 (VM-PH-18, DOC IGBRAZ), PhysicalApp_21 (VM-PH-21, DOC L2VCXO), PhysicalApp_26 (VM-PH-26, DOC UL1QUW).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Propose a maintenance window for Production apps in Maintenance: PhysicalApp_3 (8 vCPU, 8 GB; DOC FGPAA4), PhysicalApp_4 (4 vCPU, 16 GB; DOC 4MIYVZ), PhysicalApp_5 (4 vCPU, 32 GB; DOC N4DBRS), PhysicalApp_6 (8 vCPU, 8 GB; DOC 3W7MGK), PhysicalApp_8 (2 vCPU, 32 GB; DOC PPDXJX).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Create a capacity snapshot for Running/Production: PhysicalApp_19 (Storage 1261 GB; 2 vCPU, 8 GB), PhysicalApp_25 (1890 GB; 8 vCPU, 16 GB), PhysicalApp_27 (1180 GB; 4 vCPU, 8 GB), PhysicalApp_30 (991 GB; 4 vCPU, 16 GB).",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Write a cost model note proposing OpEx migration for Production apps PhysicalApp_25 (VM-PH-25) and PhysicalApp_30 (VM-PH-30), currently CapEx with Internal IT provider.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Map environments: Development — PhysicalApp_1 (Retired), _9 (Maintenance), _14 (Maintenance), _23 (Running), _28 (Maintenance); Test — PhysicalApp_7 (Maintenance), _18 (Retired), _20 (Maintenance), _21 (Retired), _24 (Running), _26 (Retired), _29 (Running); Production — multiple in Maintenance/Running as listed.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft an activation email for customer agreement CUST-0023 (Line of Credit Agreement) for Emily Williams (emily.williams@hotmail.com), Status Active, Amount 353,236.74.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "Create a termination summary for CUST-0002 (Letter of Credit) — Michael Clark (michael.clark@gmail.com), Status Terminated, Amount 499,667.34.",
    "labels": [
      "EMAIL",
      "PHONE"
    ]
  },
  {
    "text": "List active outsourcing/related supplier agreements: SUP-0012 (Outsourcing Agreement) 64,145.18; SUP-0036 (Outsourcing Agreement) 396,361.85; SUP-0039 (Outsourcing Agreement) 61,399.13; SUP-0034 (Services Supply Agreement) 237,777.45; SUP-0024 (Goods Supply Agreement) 106,623.59.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Prepare a pending-action tracker for customers: CUST-0044 (Letter of Credit) 310,738.37; CUST-0045 (Bank Guarantee) 212,569.00; CUST-0046 (Personal Loan Agreement) 451,133.43; CUST-0054 (Safe Deposit Box Agreement) 203,538.70; CUST-0055 (Credit Card Agreement) 279,479.12; CUST-0056 (Overdraft Agreement) 492,879.62.",
    "labels": [
      "PHONE"
    ]
  },
  {
    "text": "Draft a risk note flagging high amounts: CUST-0028 (Active) 497,139.00; SUP-0016 (Expired) 496,703.00; SUP-0038 (Expired) 497,735.94; CUST-0002 (Terminated) 499,667.34.",
    "labels": [
      "PHONE"
    ]
  }
]